site/ar_LB.all.json

```diff
@@ -1464,5 +1464,20 @@
   },
   "shopping_error_invalid_session_token": {
     "other": "انتهت جلسة التسوق ، يرجى المحاولة مرة أخرى."
+  },
+  "live_label_ended": {
+    "other": "مباشر: انتهى"
+  },
+  "live_label_live": {
+    "other": "يعيش"
+  },
+  "live_label_minutes": {
+    "other": "البث المباشر {{.Minutes}} دقائق من الدقائق"
```

Comment:
Don't we need more pluralities that this esp. for Arabic?

Reply:
Only need pluralities if there is a `{{.Count}}`.

Reply:
Doesn't `{{.Minutes}}` act as a de facto `{{.Count}}`? It's an integer which implies a different message in English at the very least (live in 1 minute vs. live in 2 minutes)